### PR TITLE
Ilan/withdrawable

### PR DESCRIPTION
**Pull request by ilanDoron**
please pull.

### CODE REVIEW

**Comment by ilanDoron** [comment]
@yaronvel 
 done changes. please review.

**Comment by ilanDoron** [comment]
resolve #5

File: contracts/Withdrawable.sol [hunk 1]
```diff
@@ -13,23 +13,25 @@ import "./PermissionGroups.sol";
...
-    event WithdrawToken ( ERC20 token, uint balance, address sendTo );
+    event WithdrawToken ( ERC20 token, uint amount, address sendTo );
     /**
      * @dev Withdraw all ERC20 compatible tokens
      * @param token ERC20 The address of the token contract
      */
-    function withdrawToken( ERC20 token, address sendTo ) external onlyAdmin {
+    function withdrawToken( ERC20 token, uint amount, address sendTo ) external onlyAdmin {
         uint balance = token.balanceOf(this);
-        token.transfer(sendTo, balance);
```
Comment (yaronvel) [inline contracts/Withdrawable.sol]: something wrong here. you need to transfer amount, not balance.

File: contracts/Withdrawable.sol [hunk 2]
```diff
@@ -13,23 +13,25 @@ import "./PermissionGroups.sol";
...
 
-    event WithdrawEther ( uint balance, address sendTo );
+    event WithdrawEther ( uint amount, address sendTo );
     /**
      * @dev Withdraw Ethers
      */
-    function withdrawEther ( address sendTo ) external onlyAdmin {
+    function withdrawEther ( uint amount, address sendTo ) external onlyAdmin {
         uint balance = this.balance;
+        require (amount >= balance);
```
Comment (yaronvel) [inline contracts/Withdrawable.sol]: something wrong here. you need to transfer amount, not balance.

**Comment by ilanDoron** [comment]
fixed

**Pull request merged**